packages/modelviewer.dev/src/docs-and-examples/sidebar.ts

```diff
@@ -138,6 +138,18 @@ function removeActiveEntry(sidebarIds: SidebarIds) {
   }
 }
 
+function updateHeader() {
+  // TODO, update for ids with spaces later
```

Comment:
Is this still TODO? Looks like it's working, but maybe I don't know what you mean.

Reply:
yeah, this is for categories with spaces, like Model Format, for example. I will probably do that once I start adding in more categories.